### DISCUSSION TITLE
Add colorer to text viewer/editor

### DISCUSSION THREAD

**Discussion by vasilievip** [mikekazakov/nimble-commander#176] [Ideas]
Its quite hard to view large text file w/o basic colorization

**Comment by mikekazakov** [comment]
Having a long thought about this, it looks like a doable thing.
This library can be embedded into NC for syntax parsing: 
https://github.com/ScintillaOrg/lexilla (part of SciTE:https://www.scintilla.org https://en.wikipedia.org/wiki/SciTE ).
This project looks promising from the organisation/integration viewpoint: https://github.com/magiblot/turbo.

**Comment by mikekazakov** [comment]
On track for v1.7.
